frontend/pages/index.tsx

```diff
@@ -51,9 +66,20 @@ export const getStaticProps: GetStaticProps = async () => {
       }
     }
   `;
-
+  const instagramQuery = `
```

Comment:
since this content is still within the homepage, it is better to put the studio schema still in the homePage and make a single query and then destruct the data there.